### PR TITLE
chore(profile): cleanup of profile page elements

### PR DESCRIPTION
**Pull request by jdalsem**
various improvements

1. moved title to page title
2. moved actions to title menu
3. no more special treatment (other than positioning last) for description field
4. added activity link to owner block menu (instead of action)

profiel

### CODE REVIEW

File: mod/activity/start.php [hunk 1]
```diff
@@ -29,6 +29,34 @@ function elgg_activity_page_handler($segments) {
 	return elgg_ok_response(elgg_view_resource("river", $vars));
 }
 
+/**
+ * Register menu items for the title menu
+ *
+ * @param string $hook
```
Comment (jeabakker) [inline mod/activity/start.php]: missing description including the ones below

File: mod/profile/start.php [hunk 2]
```diff
@@ -164,3 +157,39 @@ function _profile_topbar_menu($hook, $type, $return, $params) {
 	
 	return $return;
 }
+
+/**
+ * Register menu items for the title menu
+ *
+ * @param string $hook
```
Comment (jeabakker) [inline mod/profile/start.php]: missing description including the ones below

File: mod/profile/views/default/profile/details.php [hunk 3]
```diff
@@ -51,34 +45,37 @@
...
+				<div class='elgg-col elgg-col-4of5'>
+					{$field_value}
+				</div>
 			</div>
-			<?php
+___FIELD;
 		}
+		
+		$fields_output .= elgg_format_element('div', [
+			'class' => 'profile-fields',
```
Comment (jeabakker) [inline mod/profile/views/default/profile/details.php]: each field is now wrapped in `profile-fields` this only was one wrapper before. could cause problems

File: mod/profile/views/default/profile/owner_block.php [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
 /**
  * Profile owner block
+ * 
+ * @uses $vars['entity'] The user entity
  */
 
-$user = elgg_get_page_owner_entity();
-
+$user = elgg_extract('entity', $vars);
 if (!$user) {
```
Comment (jeabakker) [inline mod/profile/views/default/profile/owner_block.php]: why not check instanceof \ElggUser this covers more failures

**Review by jeabakker — DISMISSED** [review]

**Review by hypeJunction — COMMENTED** [review]
This was not a chore but a breaking change. You have removed views that are probably used by everyone with a custom profile layout. Needs to update the upgrading docs.
I'd also prefer not to sniff for page owner in title menu, instead implement  solution similar to group profiles.

**Comment by jdalsem** [comment]
Shall i create a new PR to update the docs? Or do we want this PR to be reverted?

**Comment by hypeJunction** [comment]
Updating docs should be fine I suppose

**Pull request merged**